README.md

```diff
@@ -29,13 +29,13 @@ You can run a new Electron build with this command:
 ```sh
 # The 'Hello, World!' of build-tools: get and build `main`
 # Choose the directory where Electron's source and build files will reside.
-# You can specify any path you like; this command defaults to ~/projects/electron.
+# You can specify any path you like; this command defaults to `$PWD/electron`.
 # If you're going to use multiple branches, you may want something like:
 # `--root=~/electron/branch` (e.g. `~/electron-gn/main`)
 e init --root=~/electron --bootstrap testing
 ```
 
-That command's going to run for awhile. While you're waiting, grab a
+That command's going to run for a while. While you're waiting, grab a
```

Comment:
this is not a typo: https://www.merriam-webster.com/dictionary/awhile

Reply:
@codebytere Thanks for the quick response. 
`awhile` as a word is not a typo. Its usage here can be improved by removing `for`. or use the phrase `for a while`.